pluwum/react-shoppinglist
# #152709630 user should search

#### What does this PR do?
The PR adds strictly backend search functionality with front end components rendered for testing purposes only.

#### Description of Task to be completed?
We should create A text input element that takes in string and returns values containing this string from an API to the web page.
#### How should this be manually tested?
In the running application type a text in the search box and click search. Relevant Results should appear.
#### What are the relevant pivotal tracker stories?
#152709630
